src/http_frontend/http_graph_executor_impl.cpp

```diff
@@ -86,23 +88,23 @@ Status onPacketReadySerializeImpl(
 }
 
 Status createAndPushPacketsImpl(
-    std::shared_ptr<const std::string> request,
+    std::shared_ptr<const HttpPayload> request,
     stream_types_mapping_t& inputTypes,
     PythonBackend* pythonBackend,
     ::mediapipe::CalculatorGraph& graph,
     ::mediapipe::Timestamp& currentTimestamp,
     size_t& numberOfPacketsCreated) {
     MP_RETURN_ON_FAIL(
         graph.AddPacketToInputStream(
-            "input", ::mediapipe::Adopt(new std::string(*request)).At(currentTimestamp)),
+            "input", ::mediapipe::MakePacket<HttpPayload>(*request.get()).At(currentTimestamp)),
```

Comment:
Why are we making that change? I mean using MakePacket instead of Adopt for this one.